source code/index.html

```diff
@@ -67,6 +67,11 @@ <h1>Concept of Cloud Computing</h1>
             <li>scalability to meet workload and user demands; and </li>
             <li>fewer wasted resources because customers only pay for what they use.</li>
         </ul>
+    </p>
+    <p>
+        Various Cloud Certifications for a new career in cloud computing are: 
+        <li> AWS Cloud Practioner </li>
```

Comment:
Add some source for further reading about the exams. Make sure exams are named correctly.